pubsub/CHANGELOG.md

```diff
@@ -4,6 +4,31 @@
 
 [1]: https://pypi.org/project/google-cloud-pubsub/#history
 
+## 1.0.3
+
+12-09-2019 18:51 PST
+
+### Implementation Changes
+- Update client configurations (via synth) ([#9784](https://github.com/googleapis/google-cloud-python/pull/9784))
+- Include request overhead when computing publish batch size overflow ([#9911](https://github.com/googleapis/google-cloud-python/pull/9911))
+- Split large (mod)ACK requests into smaller ones ([#9594](https://github.com/googleapis/google-cloud-python/pull/9594))
+- Fix messages delivered multiple times despite a long ACK deadline ([#9525](https://github.com/googleapis/google-cloud-python/pull/9525))
+- Update batching and flow control parameters to be same as the other client libraries ([#9597](https://github.com/googleapis/google-cloud-python/pull/9597))
+- Add 'StreamingPullManager._should_terminate' ([#9335](https://github.com/googleapis/google-cloud-python/pull/9335))
```

Comment:
(nit)
It's probably better to use backticks instead of single quotes, renders more nicely.